plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobContainer.java

```diff
@@ -78,7 +80,12 @@ public void deleteBlob(String blobName) throws IOException {
         blobStore.deleteBlob(buildKey(blobName));
     }
 
-    protected String buildKey(String blobName) {
+    @Override
+    public void deleteBlobsIgnoringIfNotExists(List<String> blobNames) throws IOException {
+        blobStore.deleteBlobs(blobNames.stream().map(this::buildKey).collect(Collectors.toList()));
```

Comment:
the method on blobstore should now also be renamed to `deleteBlobsIgnoringIfNotExists`

Reply:
done